docs/DeveloperGuide.md

```diff
@@ -2,21 +2,43 @@
 
 ## Acknowledgements
 
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links 
+to the original source as well}
 
 ## Design & implementation
 
-{Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
+The following is our class diagram for our whole project.
 
+![Local Image](UML/CLiRental.png)
+
+### Sequence diagrams
+
+1. `Adding a customer`
+
+The following sequence diagram will explain the sequence of events after the user inputs an add-user command.
+
+![Local Image](UML/AddCustomerSequence.png)
+
+2. `Creating Car file and loading file`
+
+The following sequence diagram will explain the sequence of events for loading of the carData.txt which happens at the
+start of the program. The carData.txt will be created if it does not exist at the start of program and its data will be
+loaded if the file exist.
+
+![Local Image](UML/CarFileLoader.png)
 
```

Comment:
I would consider adding a few more UML diagrams if possible. 

follow the format of the AB3 
https://se-education.org/addressbook-level3/UserGuide.html